examples/typescript-reactflux/gulpfile.js

```diff
@@ -0,0 +1,332 @@
+var require = require || null;
+
+var gulp = require('gulp');
+var del = require('del');
+var ts = require('gulp-typescript');
+var filter = require('gulp-filter');
+var log = require('gulp-print');
+var replace = require('gulp-replace');
+var reactTools = require('react-tools');
+var browserify = require('gulp-browserify');
+var concat = require('gulp-concat');
+var tsd = require('gulp-tsd');
+var closureCompiler = require('gulp-closure-compiler');
+var runSequence = require('run-sequence');
+var tslint = require('gulp-tslint');
```

Comment:
Why don't you take it easy and use a simple npm script? (this whole gulpfile, not only this piece, in fact)